src/modules/input.js

```diff
@@ -0,0 +1,89 @@
+let todos = [];
+if (localStorage.getItem('todolist') != null) {
+  todos = JSON.parse(localStorage.getItem('todolist'));
+}
+
+const saveToLocalStorage = () => {
+  localStorage.setItem('todolist', JSON.stringify(todos));
+};
+let EditTodoID = -1;
+class functio {
+    saveTodo=(todoInput) => {
+      const todovalue = todoInput.value;
+      const size = todos.length + 1;
+      const isEmpty = todovalue === '';
+
+      const isDuplicate = todos.some((todo) => todo.description === todovalue);
+      if (isEmpty) {
+        alert("Todo's input is empty");
```

Comment:
Using alternative methods to alert, you can create a more polished and professional user interface.

Reply:
@Xghayor Thanks lot for all you positive feedback